my-vault/internal/storage/migrations/000001_create_secrets_table.up.sql

```diff
@@ -0,0 +1,7 @@
+-- migrations/000001_create_secrets_table.up.sql
+
+CREATE TABLE secrets (
+    id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    name VARCHAR(255) NOT NULL,
```

Comment:
I think we can make this type `TEXT`, no? `VARCHAR` is great for limiting the size of strings but it's a bit old school.
```suggestion
    name TEXT NOT NULL,
```